README.md

```diff
@@ -1,14 +1,13 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/20ad81e6cb95ffd082d2/maintainability)](https://codeclimate.com/github/ministryofjustice/prison-visits-2/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/20ad81e6cb95ffd082d2/test_coverage)](https://codeclimate.com/github/ministryofjustice/prison-visits-2/test_coverage)
 
-# Visit someone in prison
+# Online visit requests
 
-
-A service for booking a social visit to a prisoner in England or Wales
+A staff service for reviewing social visit requests to visit a prisoner in England or Wales.
 
 ## Live application
 
-Production application is made available through GOV.UK & can be found at [https://www.gov.uk/prison-visits](https://www.gov.uk/prison-visits)
+Production application can be found at [https://staff.prisonvisits.service.justice.gov.uk](https://staff.prisonvisits.service.justice.gov.uk) accessed via DPS.
```

Comment:
Do the last 3 words make sense in this context? ...` and accessed via DPS` maybe?